app/assets/stylesheets/frontend.less

```diff
@@ -16,34 +16,43 @@ body {
 	font-family: sans-serif;
 }
 
-.section_headings {
-    position: relative;
-    z-index: 1;
-    overflow: hidden;
-    text-align: center;
+h1 {
+	/* Page itself has top padding */
```

Comment:
It's actually a standard css3 directive, see https://github.com/hotdoc/hotdoc_bootstrap_theme/commit/5f8bec4c8448d43ef2649072cc42cb009d137930 , does what you wanted a bit more generically